Cargo.toml

```diff
@@ -76,6 +76,8 @@ zip32 = "0.2"
 tonic = "0.12"
 
 [patch.crates-io]
+abscissa_core = { git = "https://github.com/str4d/abscissa.git", rev = "8da8e10cad02aa4a93099cb4942b48fbdc54eb89" }
+abscissa_tokio = { git = "https://github.com/str4d/abscissa.git", rev = "8da8e10cad02aa4a93099cb4942b48fbdc54eb89" }
```

Comment:
This is https://github.com/iqlusioninc/abscissa/pull/985.

Reply:
Reviewed that PR.